client/mass/groups.js

```diff
@@ -831,7 +827,6 @@ export function getSamplelstTW(groups, name = 'groups', notIn = true) {
 		for (const item of group.items) {
 			const value = { sampleId: item.sampleId }
 			if ('sample' in item) {
-				disabled = false
```

Comment:
this check would disable editing a group of samples that cant be seen, how is it fixed now?

Reply:
i tested and editing is still disabled correctly

Reply:
`disabled` is used in `term.groupsetting` not in `q.groupsetting{}`. `q.groupsetting` uses `.inuse`